data/json/monsters/fish.json

```diff
@@ -426,10 +426,11 @@
     "type": "MONSTER",
     "copy-from": "mon_fish_large",
     "name": { "ctxt": "fish", "str": "pike" },
-    "aggression": 10,
+    "aggression": 0,
```

Comment:
With 15 morale they tick over to hostile the moment you get in a 5-tile range, which is appropriate for territorial birds but a normal fish going "yummy" the moment someone nears the shore feels very optimistic about their abilities. After digging a bit I found about 3-4 accounts of unprovoked (as in, not in a fishing context) supposed pike attacks, otherwise I was half a mind to axe their hostility wholesale.